.github/workflows/release.yaml

```diff
@@ -2,8 +2,8 @@ name: release
 
 on:
   push:
-    branches:
-      - main
+    tags:
+      - "v*"
```

Comment:
why this change?

Right now it was pretty easy as documented at https://github.com/3scale-ops/prometheus-exporter-operator/blob/main/docs/release.md#stable, you dont need to add any manual tag 
..., you change the bundle files... and if the GH action detect there is a change on the version, execute the release process

Reply:
Release file has been removed, will be implemented in the next PR as discussed internally.